bats/tests/profile/deployment.bats

```diff
@@ -248,22 +248,21 @@ api_set() {
 }
 
 @test 'try to change locked fields via rdctl start and watch it fail' {
+    if using_dev_mode && is_macos; then
```

Comment:
Does it actually work on Linux in dev mode? Or should this be skipped in dev mode on all platforms?